src/include/dhcp-server/widgets.rb

```diff
@@ -1330,10 +1324,9 @@ def InitServiceWidget
 
       @widgets["service_status"] = {
         "widget" => :custom,
-        "custom_widget" => status_widget.widget,
-        "help"   => status_widget.help,
+        "custom_widget" => status_widget.content,
+        "help"   => "Pending",
         "init"   => fun_ref(method(:init_service_status), "void (string)"),
-        "handle" => fun_ref(method(:handle_service_status), "symbol (string, map)"),
         "store"  => fun_ref(method(:store_service_status), "void (string, map)")
       }
```

Comment:
this is wrong. Here it is CWM and old hash based one. So ideally you should here just do:

```
service_widget = CWM::ServiceWidget.new(dhcp_service)
@widgets[service_widget.widget_id] = service_widget.cwm_definitions
```
same as you can see in example for old hash based CWM.